docs/guides/admin/docs/workflowoperationhandlers/cut-marks-to-smil-woh.md

```diff
@@ -0,0 +1,57 @@
+CutMarksToSmilWorkflowOperationHandler
+===================================
+
+Description
+-----------
+
+This operation parses a JSON containing cut marks into a SMIL that can be used by the
+[VideoEditorWorkflowOperation](editor-woh.md). It does this by attributing the given times to the tracks in the
+given presentation and presenter flavors.
+
+Tracks are assumed to start at 0. Likewise, cut marks are assumed to be specified relative to the beginning
+of the tracks.
+
+
+Parameter Table
+---------------
+
+|Configuration Keys    |Example              |Description                                                    |
+|----------------------|---------------------|---------------------------------------------------------------|
+|source-media-flavors  |`presenter/prepared` |The flavors containing the video tracks.                       |
+|source-json-flavor    |`smil/times`         |The flavor of the JSON. Must contain exactly one file.         |
+|target-smil-flavor    |`smil/cutmarks`      |The flavor of the resulting SMIL.                              |
+|target-tags           |`archive`            |(Optional) Tags to add to the resulting SMIL. Default is `null`|
+
+
+JSON Format
+-----------
+
+The JSON structure specifies all segments which should be kept after cutting.
+The property `begin` marks the start of a segment while `duration` its duration.
+The values are specified in milliseconds.
+
+```json
+[
+  {
+    "begin": 1672,
+    "duration": 7199
+  }
+]
+```
+
+Operation Example
+-----------------
+
+```xml
+<operation
+    id="cut-marks-to-smil"
+    description="Process ingested cutmarks by applying them to current tracks"
+    fail-on-error="true"
+    exception-handler-workflow="partial-error">
+  <configurations>
+    <configuration key="source-media-flavors">presenter/prepared,presentation/prepared</configuration>
+    <configuration key="source-json-flavor">json/times</configuration>
```

Comment:
I'm wondering if something like `cut-marks/json` wouldn't be a better default name.

Reply:
"cut-marks" instead of "times" seems more specific and therefore clearer to understand, so +1 for that. As for the order, I put the file type first to be consistent with naming of SMIL catalogue flavors (e.g. `smil/source+partial`), but I got no hard opinion on this.

Reply:
I decided to try `cut-marks/json`. The `smil/*` is a good argument, but I don't think that smil is really meant as a file type here since it's something specific and not a general purpose file type like JSON. That would make the main type otherwise very unspecific and a wildcard like `json/*` could catch all kinds of things.

 At the same time, we also have other favors with file format as sub-type:
 
- `security/xacml(+…)`
- `captions/google-speech-json`
- `captions/ibm-watson-json`

In the end, it's just the documentation, and you can pick whatever you want anyway. Take a look, and we can still change it if you want to. I'm not that fixed on this. I will push the update shortly.